src/services/dexes/uniswapV3/UniswapV3LiquidityResolver.ts

```diff
@@ -0,0 +1,131 @@
+import {inject, injectable} from 'inversify';
+import {Logger} from 'winston';
+import {Token, ChainId} from '@uniswap/sdk-core';
+
+import TimeService from '../../TimeService.js';
+import {FeedsType} from '../../../types/Feed.js';
+import {FeedDataService} from '../../FeedDataService.js';
+import {LiquidityService} from './LiquidityService.js';
+import {LeavesAndFeeds} from '../../../types/Consensus.js';
+import {UniswapV3PoolRepository} from '../../../repositories/UniswapV3PoolRepository.js';
+import {ChainsIds} from '../../../types/ChainsIds.js';
+import {TokenRepository} from '../../../repositories/TokenRepository.js';
+import {UniswapV3Param} from './interfaces.js';
+import {DexProtocolName} from '../../../types/Dexes.js';
+import {UniswapV3Pool} from '../../../models/Pools.js';
+
+@injectable()
+export class UniswapV3LiquidityResolver {
+  @inject(FeedDataService) feedDataService!: FeedDataService;
+  @inject(TimeService) timeService!: TimeService;
+  @inject(TokenRepository) tokenRepository!: TokenRepository;
+  @inject(LiquidityService) liquidityService!: LiquidityService;
+  @inject(UniswapV3PoolRepository) uniswapV3PoolRepository!: UniswapV3PoolRepository;
+  @inject('Logger') logger!: Logger;
+
+  readonly protocol = DexProtocolName.UNISWAP_V3;
+  readonly logPrefix = '[UniswapV3LiquidityResolver]';
+
+  async apply(chainId: ChainsIds) {
+    const feeds = await this.getFeeds(chainId);
+
+    const uniswapV3Params = await this.feedDataService.getParamsByFetcherName<UniswapV3Param>(
+      feeds as LeavesAndFeeds,
+      'UniswapV3',
+    );
+
+    if (uniswapV3Params.length === 0) {
+      throw new Error(`${this.logPrefix}[${chainId}] No params for fetcher`);
```

Comment:
why error? it is ok if we don't have feeds, we simply return.

Reply:
My bad. Changed to logger.info